### PR TITLE
Azure SDK Maven archetype for creating new projects

### PR DESCRIPTION
**Pull request by srnagar**

### CODE REVIEW

**Review by JonathanGiles — APPROVED** [review]

File: packages/java-packages/azure-sdk-archetype/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Azure SDK Maven Archetype
```
Comment (weshaggard) [inline packages/java-packages/azure-sdk-archetype/README.md:1]: @srnagar please put this under the "tools" folder we are trying get all the new tools under there.

Comment (srnagar) [reply packages/java-packages/azure-sdk-archetype/README.md:1]: @weshaggard this archetype is mainly for customers of Azure SDK. This is not a tool that the Azure SDK team uses. Do you still prefer it to be in "tools" directory instead for "sdk/tools".

**Comment by weshaggard** [comment]
@srnagar just for my information why did you move to the java repo? Don't we have to build and release this before it can be used?

**Comment by srnagar** [comment]
This Maven archetype will be used by the customers of Azure SDK for Java. So, we moved it to that repo as azure-sdk-tools repo is mainly for tools used by the Azure SDK team. Yes, archetypes have to be built and released to Maven central before it can be used by customers.

**Comment by weshaggard** [comment]
OK so this is going to be another shipping library for customers then I agree it should probably go in the java repo.

**Pull request closed without merging**